Resources/doc/view_response_listener.rst

```diff
@@ -148,15 +148,16 @@ You can also define your serializer options dynamically:
 
     <?php
 
-    use FOS\RestBundle\View\View;
+    use FOS\RestBundle\Controller\Annotations\View;
```

Comment:
I agree using `ViewAnnotation`.
but `View` on this document  ( http://symfony.com/doc/current/bundles/FOSRestBundle/view_response_listener.html )
using Annotation globally, so I think remains that for consistency.
what do you think about, @Ener-Getick ?